File: Problem Set 1 - C/cash/cash.c
#include <cs50.h>
#include <math.h>
#include <stdio.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("owed change:\n ");
    }
    while (change <= 0);

    int cents = round(change * 1);
    int coins = 0;

    while (cents >= 25)
    {
        cents -= 25;
        coins++;
    }
    while (cents >= 10)
    {
        cents -= 10;
        coins++;
    }
    while (cents >= 5)
    {
        cents -= 5;
        coins++;
    }
    while (cents >= 1)
    {
        cents -= 1;
        coins++;
    }

    printf("%i", coins);
}

File: Problem Set 6 - Python/sentimental-cash/cash.py
while True:
    try:
        change = float(input("owed changes: "))
        if change > 0:
            break
    except ValueError:
        continue

cents = round(change * 100)
coins = 0

while cents >= 25:
    cents -= 25
    coins += 1

while cents >= 10:
    cents -= 10
    coins += 1

while cents >= 5:
    cents -= 5
    coins += 1

while cents >= 1:
    cents -= 1
    coins += 1

print(coins)
